packages/realm-server/package.json

```diff
@@ -97,7 +97,7 @@
     "lint:glint": "glint",
     "full-reset": "./scripts/full-reset.sh",
     "sync-stripe-products": "NODE_NO_WARNINGS=1 PGDATABASE=boxel PGPORT=5435 ts-node --transpileOnly scripts/sync-stripe-products.ts",
-    "stripe": "docker run --rm -it stripe/stripe-cli:latest"
+    "stripe": "docker run --rm --add-host=host.docker.internal:host-gateway -it stripe/stripe-cli:latest"
```

Comment:
Using `--add-host=host.docker.internal:host-gateway` ensures cross-platform compatibility for Docker container to host communication. Safe to include on all platforms as it's required for Linux and harmless on Mac/Windows